Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs

```diff
@@ -1331,6 +1336,12 @@ public Response.ManagerResponse SendAction(ManagerAction action, int timeOut)
         #endregion
 
         #region SendAction(action, responseHandler)
+        /// <summary>
+        /// Send action ans with timeout (milliseconds)

```

Comment:
Typo?

Reply:
auto comment (ans = answer) 
i'll fix

Reply:
took this out, will add it to another pull request
also took out the timeOut rename, will do that in another request too